src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('images/star-active.svg');
```

Comment:
The task specifies using a container modifier and pseudo-selectors for styling active stars, but the current implementation uses `nth-child` selectors. Consider revising this approach to align with the task requirements.